var/spack/repos/builtin/packages/mxml/package.py

```diff
@@ -37,13 +37,10 @@ class Mxml(AutotoolsPackage):
     version('2.10', '8804c961a24500a95690ef287d150abe')
     version('2.9', 'e21cad0f7aacd18f942aa0568a8dee19')
     version('2.8', 'd85ee6d30de053581242c4a86e79a5d2')
-    version('2.7', '76f2ae49bf0f5745d5cb5d9507774dc9')
-    version('2.6', '68977789ae64985dddbd1a1a1652642e')
-    version('2.5', 'f706377fba630b39fa02fd63642b17e5')
 
-    # module swap PrgEnv-intel PrgEnv-$COMP
-    # (Can use whatever compiler you want to use)
-    # Case statement to change CC and CXX flags
+    def url_for_version(self, version):
+        url = 'https://github.com/michaelrsweet/mxml/releases/download/release-{0}/mxml-{0}.tar.gz'
+        return url.format(version, version)
```

Comment:
I think I got trapped by trying to download an earlier (non-existent) version. Remove now.